cldf-metadata.json

```diff
@@ -301,7 +301,7 @@
                     }
                 ]
             },
-            "dc:extent": 8657
+            "dc:extent": 8684
         }
     ],
     "dc:bibliographicCitation": "Johann-Mattis List, Cormac Anderson, Tiago Tresoldi, & Robert Forkel. (2021). CLTS. Cross-Linguistic Transcription Systems. Zenodo. https://doi.org/10.5281/zenodo.3515744",
```

Comment:
I suggest to update this to 2023 (the DOI is the latestdoi, so no need to change here): "Johann-Mattis List, Cormac Anderson, Tiago Tresoldi, & Robert Forkel. (2023). CLTS. Cross-Linguistic Transcription Systems. Zenodo. https://doi.org/10.5281/zenodo.3515744